web/src/components/dashboard/SidebarCardsList.test.tsx

```diff
@@ -46,6 +47,30 @@ function setupMockAnalytics(): typeof analytics {
 
 const Config = getMergedConfig();
 
+const operatingPhaseWithDisplayFundingAndDisplayCertificationAreTrue = OperatingPhases.find((op): boolean => {
+  return op.displayFundings && op.displayCertifications;
+});
+
+const operatingPhaseWithDisplayCertificationIsTrue = OperatingPhases.find((op): boolean => {
+  return op.displayCertifications;
+});
+
+const operatingPhaseWithDisplayFundingIsTrue = OperatingPhases.find((op): boolean => {
+  return op.displayFundings;
+});
+
+const operatingPhaseWithDisplayCertificationAsTrueAndDisplayFundingAreFalse = OperatingPhases.find(
+  (op): boolean => {
+    return !op.displayFundings && op.displayCertifications;
+  }
+);
+
+const operatingPhaseWithDisplayFundingAndDisplayCertificationAreFalse = OperatingPhases.find(
+  (op): boolean => {
+    return !op.displayFundings && !op.displayCertifications;
+  }
+);
```

Comment:
I think I understand why you did this. Because it's actually getting the Operating Phases that will produce these cases?

Given the component doesn't require complete user data, isn't this a little overkill? I feel like the other approach was simpler to read and only defined what was absolutely necessary to run the test.

Reply:
discussed